### DISCUSSION TITLE
Slow response !

### DISCUSSION THREAD

**Discussion by badytest** [imartinez/privateGPT#244] [General]
It takes about 8 minutes to start answering my question, I mean after I had entered my query.
I am sure it will faster in the future.

**Comment by amitli1** [comment]
I also have the same slow problem.
Is there a way to check if private-gpt run on the GPU ?
What is the reasonable answering time ?

**Reply by IterNobody** [reply]
too slow. I would like to know as well.

**Comment by donedgardo** [comment]
https://www.reddit.com/r/LocalLLaMA/comments/13qo59f/how_to_increase_privategpt_performance_by_up_to_2x/
